VictoriaAlbert/MuseumObjectsTableViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  MuseumObjectsTableViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Erica Y Stevens on 11/10/16.
+//  Copyright © 2016 Erica Stevens. All rights reserved.
+//
+
+import UIKit
+
+internal let query = "vase"
+
+class MuseumObjectsTableViewController: UITableViewController {
+    
+    // MARK: Properties
+    internal let apiEndpoint = "http://www.vam.ac.uk/api/json/museumobject/search?q=\(query)"
+    internal var museumObjects = [MuseumObject]()
+    
+    // MARK: Built-In Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.title = query.uppercased()
+        
+        APIRequestManager.manager.getDataFrom(apiEndpoint: apiEndpoint) { (museumObjects: [MuseumObject]?) in
+            if museumObjects != nil {
+                DispatchQueue.main.async {
+                    self.tableView.reloadData()
+                }
+                self.museumObjects = museumObjects!
+            }
+        }
+        
+        
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return museumObjects.count
+    }
+
```

Comment:
Don't need more than one line between methods. If you want two, then it should be two everywhere.